### DISCUSSION TITLE
Post Cover Image

### DISCUSSION THREAD

**Discussion by paddy-exe** [hugo-sid/hugo-blog-awesome#11] [Feature requests]
For discoverability of specific posts perhaps a post cover image as an option with a preview in the Posts section. From Papermod theme:
![image](https://user-images.githubusercontent.com/38077837/225322339-5a18cbc1-6dfe-4711-8989-0f66821b3328.jpeg)

**Comment by DandyLyons** [comment]
hugo-blog-awesome includes Social Previews as of https://github.com/hugo-sid/hugo-blog-awesome/compare/v1.15.0...v1.16.0

In other words, when you post a link from your site on social media sites such as Reddit, then that social media site will have a preview of your link which includes a cover image. 

It might be nice for the theme to reuse some of this infrastructure for post cover images.
